packages/ui/src/components/Layout/LayoutChrome.sass

```diff
@@ -83,7 +82,7 @@
 	@media (max-width: $breakpoint-max-medium)
 		--layout-chrome-bar-width: 0
 		&-bar
-			max-width: 100vw
+			max-width: 100%
```

Comment:
proč je potřeba nastavit `max-width`? co by se stalo kdyby se to nenastavilo?

Reply:
Originally `100vw` intended to handle small screens. Long labels, wide logo would cause horizontal scrollbars. I pushed updated implementation with inner wrapper.

Small screen issue with max-width rule: | Small screen issue without max-width rule:
------|-----
<img src="https://user-images.githubusercontent.com/387611/152797218-5299b063-dd06-43c7-a53c-a89d08ba198c.png" alt="Small screen issue with max-width rule" width="320" /> | <img src="https://user-images.githubusercontent.com/387611/152797210-d4d8dc7d-f5c3-4a12-af4e-9f2ba5b57d51.png" alt="Small screen issue without max-width rule" width="320" />